assets/css/gravitee-theme.generated.css

```diff
@@ -36,7 +36,7 @@
   --gv-theme-skeleton-color: #BFBFBF;
   --gv-theme-homepage-background-color: #5A7684;
   --gv-theme-homepage-background-image: none;
-  --gv-theme-logo: url('/images/gravitee-logo.png');
-  --gv-theme-optional-logo: url('/images/gravitee-logo-light.png');
+  --gv-theme-logo: url('/images/gravitee-logo-cyan.svg');
+  --gv-theme-optional-logo: url('/images/gravitee-logo-white.svg');
```

Comment:
It's not realy necessary to change the filename ? 
if tommorow we have a yellow logo...

Reply:
As we now have the logo in 3 different colours and 2 different shapes I used a convention to avoid asking ourselves which one is which each time we use a logo. If we have to add a new colour then we will just add one named according to the colour. Or did I misunderstood your comment?